submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -47,7 +49,12 @@ open class SubmissionSubmitter(
     open fun submit(request: SubmissionRequest): Submission {
         val user = request.user.asUser()
         val extSubmission = process(request.submission, request.user.asUser(), request.files, request.method)
-        return context.saveSubmission(extSubmission, user.email, user.id).toSimpleSubmission()
+        val submitted = context.saveSubmission(extSubmission, user.email, user.id).toSimpleSubmission()
+
+        successfulSubmission.onNext(SuccessfulSubmission(
```

Comment:
I think SuccessfulSubmission can contain both SecurityUser and ExtSubmission so even is generic and we can use for other purposes in the future.